source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -594,6 +596,225 @@ For example: If there are 10 available pieces of Medium size, 15 of Large and 5
+
+<small><b><sup>1</sup></b> Beta fields are recommended to be included in the XML. However, their values
+will not be used initially and will be overridden by their respective product attributes. Also these
+fields are subject to removal from the XML feed specification.</small>
+
+#### Variation Unique ID
+
+This attribute represents the **unique** identifier of a size variation across
+shop in your database. **The value of this field must remain the same through
+a size variation's lifecycle and cannot be reused in other variations**.
+
+> A size Variation Unique ID cannot refer to **more than** one size variation. In such a
+case the size variation will be updated only the first time it's encountered in the
+data feed.
+
+> In case your e-commerce application supports more than one languages your
+data feed should only include definitions in the English language to avoid
+duplication.
+
+##### Example
+
+~~~ xml
+<variationid>12431.XL</variationid>
+~~~
+
+
+#### Variation Availability
+
+This size's variation shipping availability as used throughout your shop.
+
+<%= settings.site_name.capitalize %> uses a fixed set of availability
+descriptions that will be crosslinked to the ones provided in your feed.
+
+Available in store / Delivery 1 to 3 days
+: refers to products that are available for pick up at your outlet.
+
+Delivery 1 to 3 days
+: refers to products that can be delivered to your customer within 1 to 3 days.
+
+Delivery 4 to 10 days
+: refers to products that can be delivered to your customer within 4 to 10 days.
+
+Delivery up to 30 days
+: refers to products that can be delivered to your customer within 30 days.
+
+##### Example
+
+~~~ xml
+<availability>1 to 3 days</availability>
+~~~
+
+
+#### Variation Size
+
+If the product comes in sizes you can include all available sizes in size variations attribute.
+
+For example, a sport shoe available in 38 your `XML` will include
+the following attribute
+
+~~~ xml
+<size>38</size>
+~~~
+
+You can also use `1/2` as a half size indicator
+
+~~~ xml
+<size>38 1/2</size>
+~~~
+
+In case of clothes, sizes can be in one of the following formats:
+
+~~~ xml
+<size>XL</size>
+~~~
+
+~~~ xml
+<size>Extra Large</size>
+~~~
+
+~~~ xml
+<size>10</size>
+~~~
+
+If a size variation fits a size range (e.g. socks) then you can declare the size as a
+*range* using the dash (-). e.g.
+
+~~~ xml
+<size>5.5-6.5</size>
+~~~
+
+Finally, if a size variation is identified by two sizes you can include them using the
+forward slash. (/) e.g.
+
+~~~ xml
+<size>5.5/42</size>
+~~~
+
+> Size variations that are identified by sizes and the size information is not provided
+will **not** be classified.
+
+
+#### Variation Quantity
+
+The quantity is the amount of items available for purchase via Skroutz.
+
+> The quantity field only accepts integer values greater or equal to `0` and less or equal to `10000000`.
```

Comment:
Does 10000000 quantity make sense? We may support it internally buy maybe we should communicate a lower value to the merchants in order to "direct" them to more meaningful values... We could discuss this.